xml/admin_ceph_upgrade.xml

```diff
@@ -847,6 +847,21 @@ mds standby for name = mds.456-another-mds
       Reboot the node from the &cephos; installer DVD/image.
      </para>
     </step>
+    <step>
+      <para>
+        On the YaST command line, add the option <option>YAST_ACTIVATE_LUKS=0</option>.
+        This option ensures that no password is asked for for encrypted disks.
+      </para>
+      <warning>
+        <para>
+          This must not be enabled by default as it would break full-disk encryption
+          on the system disk or part of the system disk. This parameter only works
+          if it is provided the installer. If not provided,
+          you will be prompted for an encryption password for each individual
+          disk partition.
+        </para>
```

Comment:
Add: This is only supported since the 3rd Quarterly Update of SLES 15 SP1, so you need SLE-15-SP1-Installer-DVD-*-QU3-DVD1.iso media or newer.